shap/explainers/tree.py

```diff
@@ -46,12 +46,22 @@ class TreeExplainer(Explainer):
     implementations either inside the externel model package or in the local compiled C extention.
     """
 
-    def __init__(self, model, **kwargs):
+    def __init__(self, model, direct = False, output_type = "margin", 
```

Comment:
What's the point of direct here? It seems like a user shouldn't need to care how we parse the model.